src/pages/Contacts.jsx

```diff
@@ -35,17 +35,27 @@ const Contacts = () => {
   } = useContactsList();
   const { addNotification } = useNotification();
 
+  const getContactDisplayName = (contact) => {
+    if (!contact) {
+      return 'Unknown Contact';
+    }
+
+    const { givenName, familyName, webId } = contact;
+    const name = [givenName, familyName].filter(Boolean).join(' ');
+    return name || `Web ID: ${webId}`;
```

Comment:
For this, just the `webId` should be enough if the name is not given. Don't think we would need the extra `Web ID: ` string as part of it.